03-specifying-sample-designs.Rmd

```diff
@@ -1,5 +1,7 @@
 # Specifying sample designs and replicate weights in srvyr {#c03-specifying-sample-designs}
 
+<!-- author review -->
+
 The primary reason for using packages like {survey} and {srvyr} is to incorporate the sampling design or replicate weights into estimates. By incorporating the sampling design or replicate weights, precision estimates (e.g., standard errors and confidence intervals) are appropriately calculated. In this chapter, we will introduce common sampling designs and common types of replicate weights, the mathematical methods for calculating estimates and standard errors for a given sampling design, and the R syntax to specify the sampling design or replicate weights. While we will show the math behind the estimates, the functions in these packages will do the calculation. To deeply understand the math and the derivation, refer to @sarndal2003model, @wolter2007introduction, or @fuller2011sampling.
 
 The general process for estimation in the {srvyr} package is to:
```

Comment:
Just a note to standardize this across chapters (example: add `()` after function names)